setup.cfg

```diff
@@ -0,0 +1,30 @@
+[metadata]
+name = pyflow
+version = attr: pyflow/version.__version__
```

Comment:
Shouldn't that be `attr: pyflow.version.__version__`?

Reply:
Ops, you are right, sorry!

Reply:
Done.